CONTRIBUTING.md

```diff
@@ -60,37 +64,39 @@ into your branch:
     git checkout new_branch_name
     git merge main
     ```
-    You may need to fix merge conflicts for both of these merge
-    commands.
-1. Finally, push your change to your clone
+    You may need to fix [merge
+    conflicts](https://docs.github.com/articles/about-merge-conflicts)
+    during one or both of these merge processes.
+1. Finally, push your changes to your forked copy of the Cirq repo on GitHub:
     ```shell
     git push origin new_branch_name
     ```
-1. Now when you navigate to the cirq page on github,
-[https://github.com/quantumlib/cirq](https://github.com/quantumlib/cirq)
+1. Now when you navigate to the Cirq repository on GitHub
+([https://github.com/quantumlib/cirq](https://github.com/quantumlib/cirq)),
```

Comment:
no need for expanded `[TEXT](URL)` syntax here -
```suggestion
(https://github.com/quantumlib/Cirq),
```

Since you are at it, can you please also remove the redundant `<>` in the CLA link here -
https://github.com/quantumlib/Cirq/blob/1e4e2a70b0fd6e2219d20b1b09b435fc4ba525e6/CONTRIBUTING.md?plain=1#L17